packages/in_app_purchase/in_app_purchase_ios/ios/Classes/FIAPReceiptManager.m

```diff
@@ -9,18 +9,22 @@ @implementation FIAPReceiptManager
 
 - (NSString *)retrieveReceiptWithError:(FlutterError **)error {
   NSURL *receiptURL = [[NSBundle mainBundle] appStoreReceiptURL];
-  NSData *receipt = [self getReceiptData:receiptURL];
+  NSError *err;
```

Comment:
Please don't distinguish two otherwise identical variables by abbreviating one and not the other; it doesn't do anything to help the reader understand. Call this one `receiptDataError` or something that explains how it differs from `error`.